internal/blueprint/blueprint.go

```diff
@@ -144,9 +144,16 @@ func (b *Blueprint) CryptPasswords() error {
 	// Any passwords for users?
 	for i := range b.Customizations.User {
 		// Missing or empty password
-		if b.Customizations.User[i].Password == nil || len(*b.Customizations.User[i].Password) == 0 {
+		if b.Customizations.User[i].Password == nil {
 			continue
 		}
+
+		// Prevent empty password from being hashed
```

Comment:
Are empty passwords special in some way? Maybe add a line to explain why we don't hash them?

Reply:
Ah, sorry, I read the commit without the commit message. Do I understand correctly that an empty/unset password should mean the account is locked, but without this logic it would mean that it can be logged into with the empty password?

Reply:
Right, in my experiment I couldn't log in from the console or ssh, but as another normal user I could run 'su - user' to switch to the account with the hashed empty password.